lib/carin_for_blue_button_test_kit/custom_groups/visual_inspection_and_attestation/v200_server/11_crosscutting_group/meta_lastupdated.rb

```diff
@@ -0,0 +1,46 @@
+module CarinForBlueButtonTestKit
+  class MetaLastUpdatedTest < Inferno::Test
+    title 'Properly populates ".lastUpdated"'
+
+    description <<~DESCRIPTION
+      For
+        - C4BB-Organization
+        - C4BB-Patient
+        - C4BB-Practitioner
+        - All reference resources within a C4BB-ExplanationOfBenefit
+
+      The Health IT Module uses `.meta.lastUpdated` to mean the last time the data was updated or the date of creation in the
+      payer's system of record, whichever comes last.
+    DESCRIPTION
+
+    id :meta_last_updated
+
+    verifies_requirements 'hl7.fhir.us.carin-bb_2.0.0@89',
```

Comment:
93 listed as out of scope

Reply:
modified to be in scope. let Karl know. Will regenerate files.